Dangerfile

```diff
@@ -40,7 +40,7 @@
 # content changes within the diff
 @S_SECURITY_CONTENT_PATTERN ||= /auth|password|permission|token|secret|security|scope|api_key|apikey|sudo/
 # dont ever match against changes in these files
-@S_SECURITY_EXCLUDE_FILES ||= /test_.*\.py/
+@S_SECURITY_EXCLUDE_FILES ||= /CHANGES|tests/
```

Comment:
@mattrobenolt this is used cross-repo and it means the tests must always live in 'tests' for it to match them. Was there a reason to change the behavior there? If not we should only add CHANGES itself.